tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/deid/survey_conduct_dateshift_test.py

```diff
@@ -0,0 +1,145 @@
+"""
+Integration Test for the survey_conduct_dateshift module.
+
+Date shift any fields that are of type DATE, DATETIME, or TIMESTAMP.
+"""
+# Python imports
+import os
+from datetime import timedelta
+
+# Third party imports
+import pytz
+from dateutil import parser
+
+# Project imports
+from app_identity import PROJECT_ID
+from cdr_cleaner.cleaning_rules.deid.survey_conduct_dateshift import SurveyConductDateShiftRule
+from common import DEID_MAP
+from tests.integration_tests.data_steward.cdr_cleaner.cleaning_rules.bigquery_tests_base import BaseTest
+
+
+class SurveyConductDateShiftTest(BaseTest.DeidRulesTestBase):
+
+    @classmethod
+    def setUpClass(cls):
+        print('**************************************************************')
+        print(cls.__name__)
+        print('**************************************************************')
+
+        super().initialize_class_vars()
+
+        # set the test project identifier
+        project_id = os.environ.get(PROJECT_ID)
+        cls.project_id = project_id
+
+        # set the expected test datasets
+        dataset_id = os.environ.get('RDR_DATASET_ID')
+        cls.dataset_id = dataset_id
+        sandbox_id = dataset_id + '_sandbox'
+        cls.sandbox_id = sandbox_id
```

Comment:
I've reviewed some other PRs recently that have collapsed lines 32 and 33 into one like so:
`cls.project_id = os.environ.get(PROJECT_ID)`

And also removed string concatenation in favor of f-strings on line 38.

For consistency, could you make those few changes?